_includes/infobox_each_os.html

```diff
@@ -1,5 +1 @@
-{% if site.data.general.infobox_each_os != '' %}
```

Comment:
Isn't the content of `site.data.general.infobox_each_os` meant to control display of the content?

Reply:
Yes. You're right.